tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/AuthorityDashboard/SolutionAuthorityDashboardResultTests.cs

```diff
@@ -16,36 +17,29 @@ namespace NHSD.BuyingCatalogue.Solutions.API.UnitTests.AuthorityDashboard
     [TestFixture]
     public sealed class SolutionAuthorityDashboardResultTests
     {
-        private Mock<IMediator> _mockMediator;
-        private SolutionsController _solutionsController;
-        private const string SolutionId = "Sln1";
-
         [SetUp]
         public void SetUp()
         {
             _mockMediator = new Mock<IMediator>();
             _solutionsController = new SolutionsController(_mockMediator.Object);
         }
 
-        [Test]
-        public void NullSolutionShouldReturnEmptyAuthorityDashboardResult()
-        {
-            var dashboardAuthority = new SolutionAuthorityDashboardResult(null);
-            dashboardAuthority.Id.Should().BeNull();
-            dashboardAuthority.Name.Should().BeNull();
-            dashboardAuthority.SolutionAuthorityDashboardSections.Should().BeNull();
-        }
+        private Mock<IMediator> _mockMediator;
```

Comment:
Not sure why these fields have been moved from the top of the class?

Reply:
The standard nunit layout is setup, other members, then test methods. Can change it if we want?